streaming/base/converters/dataframe_to_mds.py

```diff
@@ -123,7 +123,7 @@ def dataframeToMDS(dataframe: DataFrame,
                    merge_index: bool = True,
                    mds_kwargs: Optional[Dict[str, Any]] = None,
                    udf_iterable: Optional[Callable] = None,
-                   udf_kwargs: Optional[Dict[str, Any]] = None) -> Tuple[Any, int]:
+                   udf_kwargs: Optional[Dict[str, Any]] = None) -> Tuple[str, str]:
```

Comment:
Checking: Is the return type always a tuple of 2 elements (`mds_path_local, `mds_path_remote`)? Or would there be a case where it's just one?

Reply:
Yes, I made it a tuple return [here](https://github.com/mosaicml/streaming/blob/d57461e64dc7a85b7aa1c8b04817b1952e93227d/streaming/base/converters/dataframe_to_mds.py#L252) for simplicity.